### DISCUSSION TITLE
Acoustic telemetry data in a winding river

### DISCUSSION THREAD

**Discussion by DEHewitt** [bmcclintock/momentuHMM#76] [Q&A] [answered]
Hi,

I have a dataset from acoustically-tagged female Giant Mud Crab (Scylla serrata) in two rivers in mid-north New South Wales (Australia). This project aims to investigate the environmental triggers of their migration to sea to spawn. The rivers are very winding (think S-shaped bends) and some of our listening stations are placed on opposite sides of a bend. 

My question is related to how suitable momentuHMM is to analyse this data given this. I have calculated the distance-to-sea of each listening station using a raster grid (size = 1m^2) and a 'least cost' path analysis. An initial idea I had was to use the difference in these distances (which account for the windiness of the river) to calculate acceleration of an individual and also the increase/decrease in their distance to sea as the data streams (instead of step length and turning angle) to infer their states.

Any advice you can provide is greatly appreciated!

Kind regards,

Dan

**Comment by DEHewitt** [comment]
I have re-read my previous post and think it needs some clarification before you can actually help. When I said "acceleration" I meant speed of a tagged individual (which can only record a detection at our fixed position receivers). Further reading suggests to me I am going to have issues with the temporal and spatial irregularity of detections, however some indication of the feasibility of such an approach would be great.

Thanks!

Dan

**Comment by bmcclintock** [comment, marked as answer]
Sorry, I'm really not qualified to provide advice on the appropriateness of these data. As you suggest, the discrete-time movement models that are implemented in momentuHMM are intended for time series of location data that are observed in regular temporal intervals and can occur anywhere in space. There probably are ways to glean similar types of inferences from acoustic tags, but I am not familiar with any such approaches.

**Comment by DEHewitt** [comment]
Understood. Thanks for your help!
